.eslintrc.yml

```diff
@@ -23,3 +23,8 @@ rules:
         - '@material-ui/core/*'
         - '@material-ui/icons/*'
         - lodash/*
+  no-console:
+    - warn
+    - allow:
+        - warn
+        - error
```

Comment:
Let's decide and do this separately from this PR

Reply:
removed